test/python/circuit/test_rust_equivalence.py

```diff
@@ -102,7 +113,7 @@ def test_matrix(self):
                 continue
 
             with self.subTest(name=name):
-                params = [pi] * standard_gate._num_params()
+                params = [0.1] * standard_gate._num_params()
```

Comment:
Was this change because `math.pi` was triggering some of the test failures? I feel like it's commonly used for assigning parameter values.

Reply:
Oh, I thought I changed it back. While debugging I did this because one of the test failures was using 0.1 as the angle value and I wanted to rule that out as the cause of the failure (checking the matrix was also correct with 0.1). I can change it back to pi, it realistically doesn't matter too much though.

Reply:
I'd even expect 0.1 to be a better value  for testing since for pi the matrices take on some special forms and 0.1 is more "irregular" in that sense 🤔

Reply:
Sounds good, I just wanted to make sure we were not ignoring pi.